poc/README.md

```diff
@@ -6,30 +6,47 @@ themesleves.
 
 ## Installation
 
-This code is compatible with SageMath version 9.6.
+This code is compatible with SageMath version 9.6. To install:
 
-In order to run the code you will need to install
-[PyCryptodome](https://pycryptodome.readthedocs.io/en/latest/index.html).
+```
+sage -pip install git+https://github.com/cfrg/draft-irtf-cfrg-vdaf@draft-irtf-cfrg-vdaf-11#subdirectory=poc
```

Comment:
Reviewer note: I've confirmed this works at cf1c27aaae0d33007e0211b6fc5b5816b98e5ed8.